NightscoutServiceKit/OTPManager.swift

```diff
@@ -74,41 +74,49 @@ public class OTPManager {
     let issuerName = "Loop"
     var tokenPeriod: TimeInterval
     var passwordDigitCount = 6
```

Comment:
Allow the maxOTPsToAccept to be parameterized. This is really just to help with testing which can be agnostic to the number of OTPs, which was not previously.